packages/portfolio-deploy/test/contract-control.test.ts

```diff
@@ -223,12 +226,16 @@ test.serial('starting while already running fails', async t => {
 });
 
 test.serial('terminate continues after error', async t => {
-  const { common, startedKitP, bundleAndInstall } = t.context;
-  const cc = await t.context.ymaxControlPK.promise;
+  const { common, bundleAndInstall, space } = t.context;
+  const { agoricNames } = common.bootstrap;
+
+  const contractKits = await space.consume.contractKits;
```

Comment:
minor: for consistent style:

```suggestion
  const { contractKits } = await space.consume;
```

Reply:
Uh that doesn't work, it's the promise inside consume that needs to be awaited

Reply:
oops. maybe I'm too tired to be reviewing this :)